Backend/db/queries.py

```diff
@@ -238,3 +233,16 @@ def update_chore(columns: list) -> str:
     for column in columns:
         query = query.set(Field(column), Parameter("?"))
     return query.get_sql()
+
+
+##########################
+#  SQL DELETE statements #
+##########################
+
+
+def delete_chore():
```

Comment:
Can replace with PyPika equivalent:
```
def delete_chore() -> str:
    query = (
        Query.from_(chores_table)
        .delete()
        .where(chores_table.ChoreId == Parameter("?"))
        .where(chores_table.ParentGoogleAccountId == Parameter("?"))
    )
    return query.get_sql()
```

Reply:
thanks, forgot to build these with pypika.

Reply:
fixed